.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Unit Tests
+
+# Run the test suite on pushes (incl. merges) to master and dev
+# Run the test suite when a PR is opened, pushed to, or reopened
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        id: setup_python_env
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          cache-dependency-path: 'luigi_pipeline/pyproject.toml'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r luigi_pipeline/requirements-dev.txt
+      - name: Check Import Sort
+        run: isort --diff --check luigi_pipeline
```

Comment:
There is an `isort` [Github action](https://pycqa.github.io/isort/docs/configuration/github_action.html) that seems reasonable.  This is one line though 🤷